IceCream/settings.py

```diff
@@ -22,7 +22,7 @@
 SECRET_KEY = 'et5csce+0jh4-8!u2s#5n!d!6nvnrs*0uw^)mgfd+(@m1wr&oz'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-# DEBUG = False
+# DEBUG = True
```

Comment:
These commented lines should be removed and the value of `DEBUG` should be taken from environment variable instead.

Reply:
@bvishal8510 Or else you can also create separate development and production settings(for eg. create a directory settings with files base.py with base settings and development.py for development settings and production.py for production settings) and set the environment variable `DJANGO_SETTINGS_MODULE` to specify the settings to be used.